examples/virtual_cluster/config/virtual_clusters/virtual-cluster-horse.yaml

```diff
@@ -0,0 +1,15 @@
+virtual_clusters:
```

Comment:
why virtual clusters are a list in the yaml?
is it for easier parsing?
```
virtual_clusters:
  - name: virtual-cluster-module-demo1
    host_cluster_name: a6-oprke2
    resources:
      max_cpu: 6
      max_mem_in_mb: 6000
      min_cpu: 2
      min_mem_in_mb: 3000
```
It’s becoming then list of lists here
```
  virtual_clusters = {
  for k in fileset("config/virtual_clusters", "virtual-cluster-*.yaml") :
  trimsuffix(k, ".yaml") => yamldecode(file("config/virtual_clusters/${k}"))
  }
```
may be will be better to flatten that

Reply:
@nikchern  as suggested removed list structure and followed same for app deploy and app profiles